src/commands/apm_cmds/version.js

```diff
@@ -1,7 +1,31 @@
-const fs = require('fs')
+const { ensureWeb3 } = require('../../helpers/web3-fallback')
+const fs = require('fs-extra')
 const findUp = require('find-up')
 const semver = require('semver')
+const APM = require('@aragon/apm')
 
+const bumpVersionAndUpdateManifest = async ({ module, bump, network, apmOptions, cwd }) => {
+
+  const web3 = await ensureWeb3(network)
```

Comment:
Yes, I considered that too. I wasn't sure if that'd be a nice DX though because it would require consumers to always create an APM instance manually first.

On the other hand, since this function is most likely be used within other commands, an APM instance prolly always available.

Reply:
When used inside the `version` command I would expect that `apm` and `web3` get initialized in the [handler](https://github.com/aragon/aragon-cli/pull/197/files/99254b4c0d321ef198a0c69d93897a34badf9111#diff-0aaf30104993c9a323278a45ec79650dR42).

Reply:
👍